core/src/main/java/hudson/Launcher.java

```diff
@@ -80,7 +80,7 @@
  *
  *
  * @author Kohsuke Kawaguchi
- * @see FilePath#createLauncher(TaskListener) 
+ * @see FilePath#createLauncher(TaskListener)
```

Comment:
It would be preferable to keep refactorings in a separate pull request. But this PR is a non-backportable feature, so I can live with that

Reply:
Sorry for that, my IDE is on steroids. I reverted the refactorings.